CONTRIBUTING.md

```diff
@@ -16,14 +16,18 @@
 ## Release Process
 
 1. Make sure the tests are passing in CI for the branch you're building
-2. Update the CHANGELOG.md with any customer-affecting changes since the last release and add this to the git index
-3. Run `npm run grunt -- release:patch` (or: "major", "minor", "patch", "prepatch") - creates commit and local tag
-4. Ensure that the infrastructure repository is in the same directory as `ably-js` and is up to date with the remote main branch
-5. Run `npm run grunt -- release:deploy` (requires the SDKDeveloper AWS role) - pushes commit and tag, then publishes to the Ably CDN
-6. Run `npm publish .` (should require OTP) - publishes to NPM
-7. Visit https://github.com/ably/ably-js/tags and add release notes to the release (generally you can just copy the notes you added to the CHANGELOG)
-8. For nontrivial releases: update the ably-js submodule ref in the realtime repo
-9. Update the Ably changelog with these changes (again, you can just copy the notes you added to the CHANGELOG)
+2. Create a new branch for the release, for example `release/1.2.3`
+3. Update the CHANGELOG.md with any customer-affecting changes since the last release and add this to the git index
+4. Run `npm version <VERSION_NUMBER> --no-git-tag-version` with the new version and add the changes to the git index
+5. Create a PR for the release branch
+6. Once the release PR is landed to the `main` branch, checkout the `main` branch locally (remember to pull the remote changes) and run `npm run build`
+7. Run `git tag <VERSION_NUMBER>` with the new version and push the tag to git
```

Comment:
I would ideally only push the tag _after_ it's been published (Ably CDN _and_ NPM)... is that possible?

Reply:
Not yet. The infrastructure script `ably-env deploy javascript` uses the tag to publish to the CDN. I agree that we should ideally push the tag after publishing but I think we deal with that when we move publishing into a CD workflow.

Reply:
Right, got you. Thanks for explaining. Will be addressed under https://github.com/ably/ably-js/issues/805.